domains/tunguyen.json

```
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "github": "tunguyen35"
```

also, please add another contact method
```suggestion
    "username": "tunguyen35"
```

---

thank you, i have reviewed and corrected